ceph/rados/core_workflows.py

```diff
@@ -1498,6 +1498,31 @@ def fetch_host_node(self, daemon_type: str, daemon_id: str = None) -> object:
             )
             return None
 
+    def fetch_host_config(self, daemon_type: str, daemon_id: str = None) -> object:
```

Comment:
Please change the method name to something unique that would reflect that the method is for `config show` command
Please create another method similar to this for `ceph daemon <> config get <param>` command